### DISCUSSION TITLE
App is not running

### DISCUSSION THREAD

**Discussion by arpit456jain** [vigneshshettyin/Flask-Generate-Certificate#197] [Q&A] [answered]
@vigneshshettyin I just pull the latest code and run the requirenents.txt successfully but on running the app I am getting an error can you please look into it.

https://user-images.githubusercontent.com/55352601/113775286-1cb41180-9746-11eb-86fe-b4f7007d53c3.mp4

**Comment by vigneshshettyin** [comment, marked as answer]
Hey, @arpit456jain we have moved from import.json to .env file as it a secure mode to store API.  Create a new file called `.env` and copy all the data from `.env.sample` to `.env` as it is. This is mentioned in installation docs as well. (https://github.com/vigneshshettyin/Flask-Generate-Certificate/blob/main/docs/INSTALLATION.md)

**Reply by arpit456jain** [reply]
Ok i will try it thanks @vigneshshettyin

**Reply by vigneshshettyin** [reply]
Cool @arpit456jain
